conda-forge.yml

```
@@ -1,13 +1,16 @@
+bot:
+  automerge: true
 build_platform:
   osx_arm64: osx_64
```

Proposing switching to cross-compilation, which works pretty well

```suggestion
build_platform:
  linux_aarch64: linux_64
  linux_ppc64le: linux_64
  osx_arm64: osx_64
```